modules/tinymce/src/plugins/spellchecker/main/ts/Plugin.ts

```diff
@@ -19,6 +19,7 @@ import * as SuggestionsMenu from './ui/SuggestionsMenu';
 
 export default () => {
   PluginManager.add('spellchecker', (editor, pluginUrl) => {
+    console.warn('The spellchecker plugin has been deprecated and marked for removal in TinyMCE 6.0');
```

Comment:
This should probably be inside the if block below, as otherwise they aren't really using it and could get confused that pro is being removed.

- [ ] Move this warning inside the if block below